test/test_core.py

```diff
@@ -3086,18 +3077,17 @@ def test_dlfcn_qsort(self):
         }
         printf("\\n");
 
-        lib_handle = dlopen("liblib.so", RTLD_NOW);
+        void* lib_handle = dlopen("liblib.so", RTLD_NOW);
         if (lib_handle == NULL) {
           printf("Could not load lib.\\n");
           return 1;
         }
-        getter_ptr = (CMP_TYPE (*)()) dlsym(lib_handle, "get_cmp");
-        if (getter_ptr == NULL) {
+        cmp_t lib_cmp = (cmp_t)dlsym(lib_handle, "lib_cmp");
+        if (lib_cmp == NULL) {
```

Comment:
These changes look separate, or am I not seeing the connection?

Reply:
I just simplified the test a little while I was trying to see what it was doing.

Reply:
Reverted these changes